ci-operator/step-registry/openshift/e2e/test/qe/disasterrecovery/openshift-e2e-test-qe-disasterrecovery-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: openshift-e2e-test-qe-disasterrecovery
+  steps:
+  - chain: cucushift-installer-check-cluster-health
+  - ref: idp-htpasswd
+  - ref: openshift-extended-test-disasterrecovery
```

Comment:
@geliu2016 I remember you need to execute case after DR.
so,
```yaml
  - ref: openshift-extended-test-disasterrecovery
  - ref: openshift-e2e-test-qe-report
```
should be
```yaml
  - ref: openshift-extended-test-disasterrecovery
  - ref: openshift-extended-test
  - ref: openshift-e2e-test-qe-report
```

Reply:
updated.

Reply:
This test chain only is for DR test cases, we don't want to include e2e test, since we hope this chain could be re-used, not only for DR CI job.

Reply:
if only executing DR case and reuse it, openshift-extended-test-disasterrecovery is ok.
no need to make openshift-e2e-test-qe-disasterrecovery chain.

you could use it in job like
```yaml
    test:
    - ref: openshift-extended-test-disasterrecovery
```

or like
```yaml
- as: vsphere-ipi-disk-encryption-tang-fips-f14
  cron: 32 20 9,23 * *
  steps:
    cluster_profile: vsphere-2
    env:
      FIPS_ENABLED: "true"
    test:
    - chain: cucushift-installer-check-cluster-health
    - ref: idp-htpasswd
    - ref: fips-check-fips-or-die
    - ref: cucushift-pre
    - ref: openshift-extended-test
    - ref: cucushift-e2e
    - ref: openshift-extended-test-supplementary
    - ref: openshift-e2e-test-qe-report
    workflow: cucushift-installer-rehearse-vsphere-ipi-disk-encryption-tang
```

you also could add it in to other chain for reuse.

the purpose we make this chain is to combine openshift-extended-test-disasterrecovery (first) and openshift-extended-test (second), so if it is not required, no need this chain and just need openshift-extended-test-disasterrecovery step.

Reply:
Here is for https://issues.redhat.com/browse/OCPQE-16831 , we need to have a consistent way, streamlined process to define some standard test suites/chains , so that it's easily and consistently to maintain them for newer and old versions  . That's why we are defining this test-chain.  So do you mean current openshift-extended-test-disasterrecovery has met this requirements?

Reply:
@wsun1 as we talked, make two chains
1, dedicated DR case chain. suggest name as openshift-e2e-test-qe-disasterrecovery
2, combined DR case and e2e chain. suggest name as openshift-e2e-test-qe-disasterrecovery-e2e (maybe you could suggest better name)

actually the only diff between them is to have openshift-extended-test or not.

@geliu2016 you could make it based on above. please reach me for unclear.

@wangke19 after the two chains are done, you need the second chain in your prow job.

Reply:
In fact, we created DR steps(- ref: openshift-extended-test-disasterrecovery) in this pr, and several steps make up the DR chain, just like: "DR steps + e2e test step = DR chain",  if DR chain only contain DR step, then DR chain is duplicated from DR steps in tech level, but as @wsun1 mentioned above"define some standard test suites/chains , so that it's easily and consistently to maintain them for newer and old versions, e2e-test chain,  olm-operator-e2e testing chain, destructive testing , long duration testing chain,  big-quota-test chain, perf-scale-test chain, stress-testing chain , reliability testing etc." So Dr test chain may work as an unify way with other test above, and it will be more easy for extending in future(such as: add some health check steps limited in DR chain only), currently, DR steps can act  as same role with DR chain without add e2e test, but DR test chain will be better from organizing and management way.

Reply:
@wsun1 @kuiwang02 I'm not sure whether we need more sync about it, both way are ok to me, and I will update the pr after you make decision.

Reply:
@kuiwang02 updated, pls take a view?